docs/components/authentication/sign-in.mdx

```diff
@@ -153,6 +153,16 @@ function mountSignIn(node: HTMLDivElement, props?: SignInProps): void;
   <!-- Add a <div id="sign-in"> element to your HTML -->
   <div id="sign-in"></div>
 
+  <!-- Initialize Clerk with your
+  Clerk Publishable key and Frontend API URL -->
+  <script
```

Comment:
Highlighted line in this mountSignIn example is off now

Reply:
Actually, all the highlighted lines on this page seem to be off

Reply:
oop i'll fix all those

Reply:
Maybe we should just drop the line highlights. I don't personally find them necessary in these specific examples, and they cost of maintaining them, in aggregate, is probably higher than the value of having them in this case, IMO.